firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/DataCollectionHelper.java

```diff
@@ -43,16 +42,10 @@
   public DataCollectionHelper(
       FirebaseApp firebaseApp,
       SharedPreferencesUtils sharedPreferencesUtils,
-      FirebaseInstanceId firebaseInstanceId,
       Subscriber firebaseEventsSubscriber) {
     this.sharedPreferencesUtils = sharedPreferencesUtils;
     isGlobalAutomaticDataCollectionEnabled =
         new AtomicBoolean(firebaseApp.isDataCollectionDefaultEnabled());
-    if (isAutomaticDataCollectionEnabled()) {
-      // Trigger this as early as possible, to minimize any latencies on returning the token
```

Comment:
Is this no longer desirable? Or did FIS change in a way that makes it a non-issue?

Reply:
This just felt like a premature optimization and it seemed like this would be a good time to understand if it was actually necessary